### PR TITLE
[MeshingApplication] Initializing data on elements and conditions on MMGProcess after remesh

### PR DESCRIPTION
**Pull request by loumalouomega**
This still doesn't solve your issue at 100% @AlejandroCornejo , Matrix™ still appears on the results

### CODE REVIEW

File: applications/MeshingApplication/custom_processes/mmg_process.cpp [hunk 1]
```diff
@@ -822,11 +820,15 @@ void MmgProcess<TMMGLibray>::ExecuteRemeshing()
         }
 
         /* We interpolate the internal variables */
-        InternalVariablesInterpolationProcess InternalVariablesInterpolation = InternalVariablesInterpolationProcess(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
-        InternalVariablesInterpolation.Execute();
+        InternalVariablesInterpolationProcess internal_variables_interpolation = InternalVariablesInterpolationProcess(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
+        internal_variables_interpolation.Execute();
```
Comment (philbucher) [inline applications/MeshingApplication/custom_processes/mmg_process.cpp:824]: Why do you call the assignemnt constructor here?

Comment (loumalouomega) [reply applications/MeshingApplication/custom_processes/mmg_process.cpp:824]: Well, it is just habit, but is not technically wrong

Comment (philbucher) [reply applications/MeshingApplication/custom_processes/mmg_process.cpp:824]: Until someone decides to delete the assignment constructor :)
(Happened to me with ModelPart)

Comment (loumalouomega) [reply applications/MeshingApplication/custom_processes/mmg_process.cpp:824]: This is a process, I will not worry about that right now

File: applications/MeshingApplication/custom_processes/mmg_process.cpp [hunk 2]
```diff
@@ -822,11 +820,15 @@ void MmgProcess<TMMGLibray>::ExecuteRemeshing()
         }
 
         /* We interpolate the internal variables */
-        InternalVariablesInterpolationProcess InternalVariablesInterpolation = InternalVariablesInterpolationProcess(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
-        InternalVariablesInterpolation.Execute();
+        InternalVariablesInterpolationProcess internal_variables_interpolation = InternalVariablesInterpolationProcess(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
```
Comment (loumalouomega) [inline applications/MeshingApplication/custom_processes/mmg_process.cpp:823]:
```suggestion
        InternalVariablesInterpolationProcess internal_variables_interpolation(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
```

Comment (loumalouomega) [reply applications/MeshingApplication/custom_processes/mmg_process.cpp:823]: @philbucher feel free of apply the change

File: applications/MeshingApplication/custom_processes/mmg_process.cpp (hunk 1 above)
Comment (philbucher) [reply applications/MeshingApplication/custom_processes/mmg_process.cpp:824]: I know, just saying
No need to change it if you know what you are doing

**Comment by loumalouomega** [comment]
Ping @AlejandroCornejo

**Review by AlejandroCornejo — APPROVED** [review]

**Comment by philbucher** [comment]
still need the branch?

**Comment by loumalouomega** [comment]
Nope

**Pull request merged**